project/BuildSettings.scala

```diff
@@ -0,0 +1,129 @@
+/**
+  * Copyright 2018 Snowplow Analytics
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *     http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
```

Comment:
doesn't scalafmt take care of properly indenting those types of comments? it seems to be

```scala
/**
 *
 */
```

across the codebase

Reply:
it seems a couple of files were not taken care of by scalafmt such as this one? not sure